docs/user/openstack/install_upi.md

```diff
@@ -885,6 +886,34 @@ The workers need no ignition override.
 
 If `os_networking_type` is set to `Kuryr` in the Ansible inventory, the playbook creates the Trunks for Kuryr to plug the containers into the OpenStack SDN.
 
+### Compute Nodes with SR-IOV NICs
```

Comment:
Would it make sense to add:

* A definition for "SR-IOV" before using the abbreviation.
* Circumstances where you would want to pay attention to this section. 
* Any differences between clusters that rely on SR-IOV and those that don't.